python/MaterialX/MANIFEST.in

```diff
@@ -0,0 +1 @@
+recursive-include libraries
```

Comment:
`setuptools` requires this. I think we should move to using that instead of `disutils.core` but for another discussion.